### DISCUSSION TITLE
ExplorerPatcher doesn't seem to work if logged in as Adminstrator

### DISCUSSION THREAD

**Discussion by MartinEB72** [valinet/ExplorerPatcher#3569] [Q&A]
While trying out a bug in Windows and/or AMD, they are saying when running when logged in as Administrator that you can get up to a 10% increase fps. Note not a user with an Admin Account, but user name Administrator. Anyways, the one thing that wouldn't work then is ExplorerPatcher. Not that I will be spending a lot of time logged in as Administrator(since it isn't very safe), but thought I would let people know

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.
Please post explorer patcher version.

**Comment by MartinEB72** [comment]
windows 11 23H2 22631.4037
ExplorerPatcher 22621.3527.65.5
Intel 9600k, so x86-64

**Comment by pyrates999** [comment]
Did you enable the account named administrator?  Normally that account is disabled.

**Comment by MartinEB72** [comment]
Yes, I had to activate that account. Like I said, it isn't what I would normally stay in since it is so vulnerable. I run a couple benchmarks to find if Intel as well as amd gets a fps boost in that mode(Intel doesn't, just AMD does). I use it strictly for testing purposes then leave. Most people would never enter it. However, something about that account will just ignore ExplorerPatcher.

**Comment by pyrates999** [comment]
Have you tried installing EP while logged into that account?

**Comment by MartinEB72** [comment]
Yes

**Comment by pyrates999** [comment]
did you try setting one or more options in the EP properties application under the user administrator?
